src/OrchardCore.Modules/OrchardCore.Security/Views/SecurityHeadersSettings.Edit.cshtml

```diff
@@ -1,11 +1,192 @@
-@using OrchardCore.Security
+@using Microsoft.AspNetCore.Mvc.Localization
+@using OrchardCore.Security.Options
 @using OrchardCore.Security.ViewModels
 @model SecuritySettingsViewModel
 
+@functions
+{
+    public LocalizedHtmlString GetContentSecurityPolicyName(string value)
+    {
+        LocalizedHtmlString name = null;
+        switch (value)
+        {
+            case ContentSecurityPolicyValue.BaseUri:
+                name = T["Base URI"];
+                break;
+            case ContentSecurityPolicyValue.ChildSource:
+                name = T["Child Source"];
+                break;
+            case ContentSecurityPolicyValue.ConnectSource:
+                name = T["Connect Source"];
+                break;
+            case ContentSecurityPolicyValue.DefaultSource:
+                name = T["Default Source"];
+                break;
+            case ContentSecurityPolicyValue.FontSource:
+                name = T["Font Source"];
+                break;
+            case ContentSecurityPolicyValue.FormAction:
+                name = T["Form Action"];
+                break;
+            case ContentSecurityPolicyValue.FrameAncestors:
+                name = T["Frame Ancestors"];
+                break;
+            case ContentSecurityPolicyValue.FrameSource:
+                name = T["Frame Source"];
+                break;
+            case ContentSecurityPolicyValue.ImageSource:
+                name = T["Image Source"];
+                break;
+            case ContentSecurityPolicyValue.ManifestSource:
+                name = T["Manifest Source"];
+                break;
+            case ContentSecurityPolicyValue.MediaSource:
+                name = T["Media Source"];
+                break;
+            case ContentSecurityPolicyValue.ObjectSource:
+                name = T["Object Source"];
+                break;
+            case ContentSecurityPolicyValue.Sandbox:
+                name = T["Sandbox"];
+                break;
+            case ContentSecurityPolicyValue.ScriptSource:
+                name = T["Script Source"];
+                break;
+            case ContentSecurityPolicyValue.StyleSource:
+                name = T["Style Source"];
+                break;
+        }
+
+        return name;
+    }
+
+    public LocalizedHtmlString GetPermissionPolicyName(string value)
```

Comment:
Note all of them are here. For the full list see: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Feature-Policy#directives E.g. and `gamepad` and `document-domain` are not here.

Reply:
This is the browser compatibility of permissions policies https://developer.mozilla.org/en-US/docs/Web/HTTP/Feature_Policy#browser_compatibility, some of them are experimental (expect behavior to change in the future.) So can we include them or shall list the others.

FYI you may some warnings in some browsers because they are incompatible

Reply:
Since the list this module uses is not extendable from the UI (i.e. if you want to use other CSP headers you have to write code), I'd include all of them.

Reply:
Make the UI extensible may it needs an extra works this could be done after this PR for now we could include them